tf2onnx/graph.py

```diff
@@ -1193,10 +1193,8 @@ def make_model(self, graph_doc, optimize=False, graph_name="tf2onnx", external_t
                       "producer_version": __version__}
 
         if "opset_imports" not in kwargs:
-            opsets = []
-            imp = OperatorSetIdProto()
-            imp.version = self._opset
-            opsets.append(imp)
+            opsets = [helper.make_opsetid(constants.ONNX_DOMAIN, self._opset)]
+            opsets.append(constants.AI_ONNX_ML_OPSET)
```

Comment:
Add a condition to check if `AI_ONNX_MLDOMAIN opset 2` in list `opsets`?
And the latest `ai.onnx.ml` is `3`, should we also add it? https://github.com/onnx/onnx/blob/main/docs/Versioning.md#released-versions

Reply:
ORT doesn't support 3 for this domain yet, so 2 is good enough.